pkg/api/dashboard.go

```diff
@@ -295,8 +296,16 @@ func (hs *HTTPServer) PostDashboard(c *models.ReqContext, cmd models.SaveDashboa
 			return response.Error(500, "Error while applying default value to the dashboard json", err)
 		}
 	}
+	if cmd.FolderUid != "" {
```

Comment:
Do we need to validate if both folderId and folderUid is set? Or is the idea that if folderUid is set it will override folderId?

Reply:
FolderUID and FolderID both present - is a confusing combination. We could either a) treat it as an error or b) make one of the values override the other. In case of b) I would prefer UID to override ID, assuming that IDs are to be gone in the future.

Reply:
I agree with your suggestion of b)